Gems/PhysX/Code/Source/EditorColliderComponent.cpp

```diff
@@ -768,33 +773,49 @@ namespace PhysX
         {
             m_componentWarnings.clear();
             m_configuration.m_materialSelection.SetMaterialSlots(Physics::MaterialSelection::SlotsArray());
+            AzToolsFramework::ToolsApplicationEvents::Bus::Broadcast(&AzToolsFramework::ToolsApplicationEvents::InvalidatePropertyDisplay, AzToolsFramework::Refresh_EntireTree);
```

Comment:
nit: Please add a line break after the first paren (to make the line not quite so long)
e.g.
```
AzToolsFramework::ToolsApplicationEvents::Bus::Broadcast(
    &AzToolsFramework::ToolsApplicationEvents::InvalidatePropertyDisplay, AzToolsFramework::Refresh_EntireTree);
```